indico/modules/events/registration/controllers/__init__.py

```diff
@@ -30,21 +32,17 @@ def _process_args(self):
 
 
 class RegistrationEditMixin:
-    def _process(self):
-        form = make_registration_form(self.regform, management=self.management, registration=self.registration)()
-
-        if form.validate_on_submit():
-            data = form.data
-            notify_user = not self.management or data.pop('notify_user', False)
-            if self.management:
-                session['registration_notify_user_default'] = notify_user
-            modify_registration(self.registration, data, management=self.management, notify_user=notify_user)
-            return redirect(self.success_url)
-        elif form.is_submitted():
-            # not very pretty but usually this never happens thanks to client-side validation
-            for error in form.error_list:
-                flash(error, 'error')
+    def _process_POST(self):
+        schema = make_registration_schema(self.regform, management=self.management, registration=self.registration)()
+        form = parser.parse(schema)
```

Comment:
I'd rename this to `data` or `form_data`... (a variable named `form` is pretty much always a WTForm in our codebase)